server/src/routes/api/v1/votesRouter.js

```diff
@@ -0,0 +1,58 @@
+import express from 'express'
+import { Vote } from '../../../models/index.js'
+import VoteSerializer from '../../../../serializers/VoteSerializer.js'
+import { Review } from '../../../models/index.js'
+
+const votesRouter = new express.Router()
+
+votesRouter.get('/:reviewId', async (req, res) => {
+  try {
+    const { reviewId } = req.params
+    const totalCount = await VoteSerializer.getTotal(reviewId)
+    return res.status(201).json({ totalCount })
+  } catch (error) {
+    console.log(error)
+    return res.status(500).json({ errors: error })
+  }
+})
+
+votesRouter.post('/:reviewId', async (req, res) => {
+  try {
+    const { voteValue } = req.body
+    const { reviewId } = req.params
+    const priorVote = await VoteSerializer.checkForPriorVote(reviewId, req.user.id)
+    if (!priorVote) {
+      await Vote.query().insertAndFetch({
+        value: voteValue,
+        userId: req.user.id,
+        reviewId: reviewId
+      })
+      const totalCount = await VoteSerializer.getTotal(reviewId)
+      return res.status(201).json({ totalCount })
+    } else {
+      return res.status(422).json({ message: "You already voted" })
```

Comment:
We might consider combining the functionality of the POST and DELETE endpoints to be able to achieve toggling between selection a vote option, that way if a user clicks on the up vote, and up vote again, it will remove their vote instead of giving a message of "you already voted"